### PR TITLE
Add checkbox option in context menu for tab closing behavior

### PR DESCRIPTION
**Pull request by frankyjuang**
#25 
@orschiro you might want to refine the option name which I'm not quite satisfied with 🙃

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
 let currentTabId;
-let currentTabJustRemoved = false;
+let currentTabJustRemoved = false; // No-use if `defaultTabClosingBehavior`.
+let defaultTabClosingBehavior = false;
 
 function init() {
 	tabHistory[-1] = {id: null}; // Dummy start.
+	chrome.contextMenus.create({
+		type: "checkbox",
+		title: "Go to last tab after closing current one",
```
Comment (orschiro) [inline src/background.js]: I think we should make this as explicit and clear as possible: 

"Go to previously viewed tab after closing current one"

Do you agree?

**Comment by frankyjuang** [comment]
What was the original behavior we had for this plugin? I'm thinking of setting this option checked as default or not.

File: src/background.js (hunk 1 above)
Comment (frankyjuang) [reply src/background.js]: Great!

**Comment by orschiro** [comment]
@frankyjuang default was NOT going back to the previously viewed tab which I think we should maintain that way and follow Chrome's default behaviour unless people want to tweak using this extension.

**Pull request merged**